### PR TITLE
Fix/edit user label

### PR DESCRIPTION
**Pull request by agagancarczyk**
## Motivation
Fix for #1627

## Verification Steps
1. Go to users menu item, credentials tab and set up a new credential.
2. User label should be empty by default, select a pencil icon and enter a text in the text box and select a tick icon.
3. Observe the user label populated with the correct text on the first try. Edit a few times to double-check.

## Checklist:

- [ ] Code has been tested locally by PR requester
- [ ] User-visible strings are using the react-i18next framework (useTranslation)
- [ ] Help has been implemented
- [ ] axe report has been run and resulting a11y issues have been resolved
- [ ] Unit tests have been created/updated

### CODE REVIEW

File: src/user/UserCredentials.tsx [hunk 1]
```diff
@@ -243,26 +243,31 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
...
-        },
-        userLabelFormValue.userLabel || ""
-      );
-      refresh();
-      addAlert(t("updateCredentialUserLabelSuccess"), AlertVariant.success);
-      setIsUserLabelEdit(false);
-    } catch (error) {
-      addError(t("updateCredentialUserLabelError"), error);
-      setIsUserLabelEdit(false);
+    } else {
```
Comment (jonkoops) [inline src/user/UserCredentials.tsx:254]: No need to do an else here as the `if` clause is terminating due to the `return` statement.

File: src/user/UserCredentials.tsx [hunk 2]
```diff
@@ -243,26 +243,31 @@ export const UserCredentials = ({ user }: UserCredentialsProps) => {
...
+          },
+          userLabelFormValue.userLabel || ""
+        );
+        refresh();
+        addAlert(t("updateCredentialUserLabelSuccess"), AlertVariant.success);
+        setIsUserLabelEdit(false);
+        setEditedUserCredential({});
+      } catch (error) {
+        addError(t("updateCredentialUserLabelError"), error);
+        setIsUserLabelEdit(false);
```
Comment (jonkoops) [inline src/user/UserCredentials.tsx:269]: Since this is executed both in the `try` and `catch` clauses, you can move it outside of them.

**Review by jonkoops — CHANGES_REQUESTED** [review]

**Comment by agagancarczyk** [comment]
Thanks @jonkoops - this is now ready for a final check!

**Review by jonkoops — APPROVED** [review]
LGTM

**Pull request merged**